test/prototype_transforms_kernel_infos.py

```diff
@@ -348,17 +355,22 @@ def reference_resize_bounding_box(bounding_box, *, spatial_size, size, max_size=
     )
 
     expected_bboxes = reference_affine_bounding_box_helper(
-        bounding_box, format=datapoints.BoundingBoxFormat.XYXY, affine_matrix=affine_matrix
+        bounding_box, format=format, spatial_size=spatial_size, affine_matrix=affine_matrix
```

Comment:
The problem is here. You are passing wrong output size, it should be 
```suggestion
        bounding_box, format=format, spatial_size=(new_height, new_width), affine_matrix=affine_matrix
```